src/components/Add.js

```diff
@@ -95,7 +95,7 @@ const Add = () => {
           id="species"
           type="text" 
           required
-          pattern="[a-zA-Z0-9]+"
+          pattern="[a-zA-ZåäöÅÄÖ0-9]+"
```

Comment:
Doing the same changes in more than one place usually mean that you should lift out the input field and create a AddEditInput that take the onChange as input.